### ISSUE TITLE
Add timestamp to explosions.log

### ISSUE THREAD

**Issue by YanFett** [GTNewHorizons/GT-New-Horizons-Modpack#9558]
### Your GTNH Discord Username

reverse71

### Your Pack Version

2.1.2.1

### Your Proposal

explosions.log should have timestamps added to each explosion.
Also bombcar said it should rotate.

### Your Goal

to better understand what explosion happened when, when searching for the source.

### Your Vision

When explode you understand.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by bombcar** [comment]
We delete or overwrite this log so by the time people ask in chat it is often too late.

We should either keep the last five explosions or log to the file and linerotate it or ?

**Comment by YanFett** [comment]
By last five explosions do you mean the last 5 large explosion events or litterally the last 5 machines that blew up?
My last explosion included about 20 machines, so having it overwrite inbetween would not be helpful.

**Comment by bombcar** [comment]
Last five “events” is what I would aim for - not being able to track down what happened is annoying.

**Comment by Savallator** [comment]
5 feels a bit arbitrary here... i mean, we are no longer in 1999, we can easily have it save the last 100 or 1000 Explosions without any impact on disk usage...

**Comment by bombcar** [comment]
yeah, the proper is a log rotate setup but in reality if your explosions.log fills your disk and crashes your server, you had other problems.
